packages/core/src/layout/HeaderTabs/index.tsx

```diff
@@ -46,15 +46,24 @@ export type Tab = {
 export const HeaderTabs: React.FC<{
   tabs: Tab[];
   onChange?: (index: number) => void;
-}> = ({ tabs, onChange }) => {
-  const [selectedTab, setSelectedTab] = useState<number>(0);
+  selectedIndex?: number;
+}> = ({ tabs, onChange, selectedIndex }) => {
+  const [selectedTab, setSelectedTab] = useState<number>(selectedIndex ?? 0);
```

Comment:
Do these tabs offer the ability to BOTH support active and passive mode so to speak? I was glad to see the `selectedIndex` prop, and I think that if we can get away with it, we should use that strategy everywhere (forcing the caller to manage the tab state). Does that make sense? Then we would remove this `selectedTab` state entirely and simplify this component a bit.

I won't block this PR on that, but it should probably be the "way of the future" :)

Reply:
Yea, this makes it support both. Settling on one way would be nice, but i'll leave this here for now. No idea what would break if i removed it's own state management.